servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/gc/AbstractRestGCTest.java

```diff
@@ -0,0 +1,370 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.jaxrs.gc;
+
+import java.time.Instant;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+import org.projectnessie.error.BaseNessieClientServerException;
+import org.projectnessie.model.Branch;
+import org.projectnessie.model.Content;
+
+public abstract class AbstractRestGCTest extends AbstractRestGC {
+
+  @Test
+  public void testSingleRefMultiTable() throws BaseNessieClientServerException {
+    String prefix = "singleRefMultiTable";
+    Map<String, Map<String, Set<Content>>> expectedExpired = new HashMap<>();
+
+    Branch branch1 = createBranch(prefix);
+    // one commit for "table_1" on branch1
+    CommitOutput table1 =
+        commitSingleOp(
+            prefix, branch1, branch1.getHash(), 42, "cid_1", "table_1", "file1", null, null);
+    // two commits for "table_2" on branch1
+    CommitOutput table2 =
+        commitSingleOp(
+            prefix, branch1, branch1.getHash(), 42, "cid_2", "table_2", "file1", null, null);
+    table2 =
+        commitSingleOp(
+            prefix, branch1, table2.hash, 43, "cid_2", "table_2", "file2", table2.content, null);
+    // All 3 commits are expected to be expired as it is before the cutoff time and not the head
+    // commits.
+    fillExpectedContents(branch1, 4, expectedExpired);
```

Comment:
The comment says *3* commits, but here there are *4* commits. That's confusing.

Reply:
it should be 3 commits in the arguments. But there are totally 3 commits itself. So no functional impact. I will fix it.